config/locales/devise.en.yml

```diff
@@ -75,7 +75,7 @@ en:
       last_name: "Last Name:*"
       first_name: "First Name:*"
       company: "Company:"
-      email: "Email:"
+      email: "Email:*"
```

Comment:
Sounds good. I'm going to leave this PR how it is for now, but I'm going to put a story in to do this throughout the app.